doltcli/dolt.py

```diff
@@ -1345,7 +1345,7 @@ def ls(self, system: bool = False, all: bool = False, **kwargs) -> List[TableT]:
                 if not line:
                     pass
                 split = line.lstrip().split()
-                tables.append(Table(name=split[0], root=split[1], row_cnt=int(split[2])))
+                tables.append(Table(name=split[0], row_cnt=int(split[1])))
```

Comment:
if we did a switch on the length of split, or had `row_cnt=int(split[len(split)-1])`, then this would be backwards compatible

Reply:
Okay sounds good